### ISSUE TITLE
defenders of darrowshire - spirits freed is not party wide. the ...

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#7527]
> defenders of darrowshire - spirits freed is not party wide. the progress is not shared

.go xyz 2930.340000 -3586.860000 115.613000 0 4.341210

**Comment by Bennylavaa** [comment]
This is true, not sure if it should be shared or not though seeing you have to talk to the npc to get credit.
